src/Vencord.ts

```diff
@@ -30,17 +30,34 @@ import "./webpack/patchWebpack";
 import { popNotice, showNotice } from "./api/Notices";
 import { PlainSettings, Settings } from "./api/settings";
 import { patches, PMLogger, startAllPlugins } from "./plugins";
+import { getCloudSettings, shouldCloudSync } from "./utils/settingsSync";
 import { checkForUpdates, rebuild, update, UpdateLogger } from "./utils/updater";
 import { onceReady } from "./webpack";
 import { SettingsRouter } from "./webpack/common";
 
 export let Components: any;
 
+async function syncSettings() {
+    if (Settings.backend.settingsSync && await shouldCloudSync()) {
+        await getCloudSettings(false);
+
+        showNotice(
+            "Vencord settings have been updated from the cloud!",
+            "Restart",
+            () => {
+                location.reload();
+            }
+        );
+    }
+}
+
 async function init() {
     await onceReady;
     startAllPlugins();
     Components = await import("./components");
 
+    await syncSettings();
```

Comment:
I don't think blocking here is relevant
```suggestion
    syncSettings();
```